mantle/cmd/kola/qemuexec.go

```diff
@@ -144,9 +147,39 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if directIgnition && ignition == "" {
-		return fmt.Errorf("Cannot use ignition-direct without a path to an Ignition config")
+	if ignition != "" && butane != "" {
+		return fmt.Errorf("Cannot use both --ignition and --butane")
 	}
+	if directIgnition && ignition == "" && butane == "" {
+		return fmt.Errorf("Cannot use --ignition-direct without --ignition or --butane")
+	}
+	if len(ignitionFragments) > 0 && directIgnition {
+		return fmt.Errorf("Cannot use --add-ignition with --ignition-direct")
+	}
+	if len(bindro) > 0 && directIgnition {
+		return fmt.Errorf("Cannot use --bind-ro with --ignition-direct")
+	}
+	if len(bindrw) > 0 && directIgnition {
+		return fmt.Errorf("Cannot use --bind-rw with --ignition-direct")
+	}
+
+	builder := platform.NewBuilder()
+	defer builder.Close()
+
+	if butane != "" {
+		tmpf, err := builder.TempFile("ignition")
+		if err != nil {
+			return err
+		}
+		defer tmpf.Close()
+		cmd := exec.Command("fcct", butane)
```

Comment:
Why shell out rather than vendoring the package?

Reply:
Because vendoring would require me to look up things and work with go modules. :stuck_out_tongue: 

But more seriously, I can do this if there's a strong preference but meh, cosa is defined as a container and we ship fcct/butane so we'd be fattening our go binaries for not much gain.

Reply:
I'm definitely in favor of execing binaries where it makes sense, and I agree it does here.

Reply:
Yeah, I'm generally in favor of not execing binaries when it's avoidable, but I'm okay with this for now.  I'll just note that this continues down the road of making kola hard to use outside the container.